robot_skills/src/robot_skills/robot.py

```diff
@@ -131,16 +131,30 @@ def reset(self):
                 bodypart.reset()
         return all(results.values())
 
+    def reset_all_arms(self, arm_timeout=0.0, close_gripper=True, gripper_timeout=None):
```

Comment:
If I am correct the arm_time follows the following path:
arm.reset --> arms.reset --> send_joint_goal --> send_joint_trajectory --> goal_time_tolerance.

The goal_time_tolerance should not be 0.0 seconds by default right?

Reply:
Looks like it, but it there is no explanation of what the timeout is about. I grabbed `0.0` from the `Arm.reset` function, lower level functions seem to have other defaults for unknown reason.

In all checked out repositories at my system (but that is incomplete), the only other mention of `goal_time_tolerance` is in `control_msgs/control_msgs/action/FollowJointTrajectory.action`:
```
# To report success, the joints must be within goal_tolerance of the
# final trajectory value.  The goal must be achieved by time the
# trajectory ends plus goal_time_tolerance.  (goal_time_tolerance
# allows some leeway in time, so that the trajectory goal can still
# succeed even if the joints reach the goal some time after the
# precise end time of the trajectory).
#
# If the joints are not within goal_tolerance after "trajectory finish
# time" + goal_time_tolerance, the goal aborts with error_code set to
# GOAL_TOLERANCE_VIOLATED
```
That reads as additional time on top of the normally required time. For a reliable system, `0.0` should be fine in that case.

Reply:
This description indeed seems to indicate that a default of 0.0 is fine!